autoscale-email-alert-dispatcher/src/main/java/com/github/autoscaler/autoscale/email/alert/dispatcher/EmailDispatcher.java

```diff
@@ -86,14 +86,14 @@ public void dispatch(final String emailBody) throws ScalerException
 
             // Send the message.
             try {
-                LOG.debug("Sending email...");
+                LOG.info("Sending email...");
```

Comment:
I don't think the sending of email should be logged at INFO level. @dermot-hardy What do you think?

Reply:
Yeah I think you're right, since the "Email sent!" is now at info, and it does log if the e-mail is not sent, it would probably just be duplicate logging.

Reply:
@rado-straka Can you change this one back to DEBUG. the rest look fine to me.